big_tests/tests/sm_SUITE.erl

```diff
@@ -847,15 +844,13 @@ subscription_requests_are_buffered_properly(Config) ->
         % ...and reconnects with session replacement.
         {ok, Alice2, _} = escalus_connection:start(AliceSpec),
 
-        % THEN Alice receives (without sending initial presence):
+        % THEN Alice DOES NOT receive anything, including
         % * buffered available presence (because it's addressed to full JID)
         % * buffered Bob's message (like above)
-        % Alice DOESN'T receive:
         % * buffered subscription request because it is dropped by ejabberd_sm
         %   because it's treated like repeated sub request to bare JID, so it's not
         %   processed by any sub req handler (like mod_roster) 
-        SubReqAndInitialPresence = escalus:wait_for_stanzas(Alice2, 2),
-        escalus:assert_many([AvailablePredFun, MsgPredFun], SubReqAndInitialPresence),
+        false = escalus_client:has_stanzas(Alice2),
```

Comment:
again, why is it just one stanza now?

Reply:
It is zero stanzas that the user is supposed to receive, this is a session replacement, not a resumption. If the user wants old stuff, he should do resumption, or request archive, or offline messages, or anything like that. In this case, the user is plainly replacing an old session.